index.rst

```diff
@@ -11,14 +11,78 @@ Bharat-Sim's documentation!
    :caption: Contents:
 
 
-
-Indices and tables
-==================
-
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
 
-How does this lool?
+
+
+
+SIR Model with Agents
```

Comment:
This is not quite what I meant by the "SIR Model with Agents". Go over the notes that I had sent you. This section should not rely on BharatSim at all, as it's part of the Epidemiology section. It has more to do with how one would model an SIR-type system using a stochastic agent-based scheme, like I described in the notes I sent you. 

This section should be renamed, it is actually more similar to the "Step-by-step construction of the SIR model in BharatSim" that @prathithbhargav  is supposed to be working on. However, it will still require a little reworking, since it doesn't describe the construction of the model step-by-step.

Reply:
Okay, I think I understand. So initially, the population is assumed to be well-mixed when we get the deterministic solution. When we switch to a population that is not well-mixed, we start getting stochastic solutions. Do I explain the part under "Stochastic solutions to this model" in your notes, because that part explains the role of agents?

Reply:
Yes, that's basically what I meant. This section is an aid to people who have very little knowledge of how to actually model simple epidemiological systems. My notes should be a good skeleton, over which you can put in anything you feel could've been explained better.

Reply:
Alright, sure